jpo-ode-svcs/src/test/java/us/dot/its/jpo/ode/OdePropertiesTest.java

```diff
@@ -80,13 +70,8 @@ public void testSettersAndGetters() {
             int testTrustRetries = 23;
             String testKafkaTopicOdeBsmPojo = "testKafkaTopicOdeBsmPojo";
             String testKafkaTopicOdeBsmJson = "testKafkaTopicOdeBsmJson";
-            String testVersion = "1.1.0-SNAPSHOT";
```

Comment:
note: since version is provided by BuildProperties it doesn't seem worth our time to test that the provided library does what it says it does